.github/workflows/ci-macos.yml

```diff
@@ -99,61 +99,3 @@ jobs:
         with:
           path: "dist/*.tar.gz"
           name: dist
-
-  local-macos-nohomebrew:
-
-    needs: [dist]
-
-    runs-on: ${{ matrix.os }}
```

Comment:
Is this also failing? I cannot find a run.

Reply:
Yes, all of them are failing see eg https://github.com/sagemath/sage/actions/runs/12979684206/job/36196002577